src/app/image-slots/image-slots.component.scss

```diff
@@ -0,0 +1,52 @@
+.image-wrapper {
+
+}
+
+.img-box {
+
+    //border-radius: 50px;
+    border-radius: 10%;
+    overflow: hidden;
+    position: relative;
+    height: 17rem;
+    width: 15%;
+    padding: 0;
+}
+
+.img-box-selected {
+    width: 28%;
+}
+.img-box i {
+    position: absolute;
+    left:50%;
+    bottom: 10px;
+    transform: translateX(-50%);
+    background-color: white;
+    padding: 6%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 100%;
+}
+.img-box:nth-child(1) {
+    background: url(https://picsum.photos/500);
+}
+.img-box:nth-child(2) {
+    background: url(https://picsum.photos/500);
+}
+.img-box:nth-child(3) {
+    background: url(https://picsum.photos/500);
+}
+.img-box:nth-child(4) {
+    background: url(https://picsum.photos/500);
+}
+.img-box:nth-child(5) {
+    background: url(https://picsum.photos/500);
+}
+.img-box:nth-child(6) {
+    background: url(https://picsum.photos/500);
```

Comment:
Jeżeli korzystasz z tego zdjęcia tyle, to powinno to być zapisane w jakiś inny sposób żebyś tego nie musiał tyle razy kopiować, chyba że zamysł jest tutaj taki że były by różne zdjęcia, to wtedy spoko, chociaż i tak chyba bym wtedy zrobił to że w innym miejscu daje się te zdjęcia a nie w css